packages/code-du-travail-api/src/server/routes/items/relatedItems.elastic.js

```diff
@@ -4,7 +4,7 @@ function getRelatedItemsBody({ settings, size = 10, sources = [] }) {
   }
   return {
     size,
-    _source: ["title", "source", "slug"],
+    _source: ["action", "description", "icon", "title", "source", "slug"],
```

Comment:
do we need description in related Items ? 
![image](https://user-images.githubusercontent.com/160320/70738186-87049f80-1d14-11ea-909c-b3c4d7666fc6.png)

Reply:
<img width="1304" alt="Screenshot 2019-12-12 at 20 00 58" src="https://user-images.githubusercontent.com/705453/70740745-24160700-1d1a-11ea-9d5d-7093850b1390.png">
C'est dans le design !